src/app/shared/recognize-video-type-id.ts

```diff
@@ -0,0 +1,199 @@
+
+const getSrc = require('get-src');
+
+export function getVideoId(str: any): any {
```

Comment:
no implicit any tzn jeśli już używasz any w TS to pisz komentarz dlaczego (podpowiedź - tutaj to nie ma sensu :p musisz napisać że parametr który przyjmuje fuinkcja to string a z funkcji wychodzi pewien obiekt. Interfejs tego obiektu możesz sobie napisać w tym pliku lub dać do interfaces, oba miejsca mają sens

Reply:
a naming tego interrfejsu skoro gdzie indziej nazywasz to metadada może być np MovieMetadata